src/Servers/Kestrel/Core/src/Middleware/Http3ConnectionMiddleware.cs

```diff
@@ -25,18 +24,19 @@ public Task OnConnectionAsync(MultiplexedConnectionContext connectionContext)
         {
             var memoryPoolFeature = connectionContext.Features.Get<IMemoryPoolFeature>();
 
-            var http3ConnectionContext = new Http3ConnectionContext(
+            var httpConnectionContext = new Http3ConnectionContext(
```

Comment:
Is it possible to remove this middleware now?

Reply:
I'm going to stick with what connection logging middleware does and have separate middleware. I'll rename this one to multiplex.